ide/preferences.ml

```diff
@@ -359,14 +366,22 @@ let text_font =
   in
   new preference ~name:["text_font"] ~init ~repr:Repr.(string)
 
+let is_standard_doc_url url =
+  let wwwcompatprefix = "http://www.lix.polytechnique.fr/coq/" in
```

Comment:
Since you are moving this function around, I think it is time to remove this compat thing and simplify the code.

Reply:
Sure, I planned to do that however I got scared by the logic there.